yarn.lock

```diff
@@ -2908,21 +2895,21 @@
     debug "^4.3.4"
     typeorm "^0.3.10"
 
-"@sphereon/ssi-sdk-data-store-common@file:../ssi-sdk/packages/data-store-common/sphereon-ssi-sdk-data-store-common-v0.9.9-bram.tgz":
-  version "0.9.9-bram"
-  resolved "file:../ssi-sdk/packages/data-store-common/sphereon-ssi-sdk-data-store-common-v0.9.9-bram.tgz#c0a860d691fd76d7f3106013f47cca1d32152fbc"
+"@sphereon/ssi-sdk-data-store-common@file:../ssi-sdk/packages/data-store-common/sphereon-ssi-sdk-data-store-common-v0.9.15-bram.tgz":
```

Comment:
This will be problematic for other devs

Reply:
i need to do a yarn install anyways for the new artefacts. Thats why i did not bother to remove these

Reply:
will be fixed once WAL-365 can be merged (which should have this fixed by then)